docs/architecture/modernize-with-azure-containers/walkthroughs-technical-get-started-overview.md

```diff
@@ -14,7 +14,7 @@ The following sections explain what each walkthrough is about, its objectives an
 
 The following get-started walkthroughs provide consistent and comprehensive technical guidance for sample apps that you can lift and shift by using containers, and then move by using multiple deployment choices in Azure.
 
-Each of the following walkthroughs uses the new sample eShopLegacy and eShopModernizing apps, which are available on GitHub at <https://github.com/dotnet-architecture/eShopModernizing>.
+Each of the following walkthroughs uses the new sample `eShopLegacy` and `eShopModernizing` apps, which are available on GitHub at <https://github.com/dotnet-architecture/eShopModernizing>.
```

Comment:
```suggestion
Each of the following walkthroughs uses the new sample eShopLegacy and eShopModernizing apps, which are available on GitHub at <https://github.com/dotnet-architecture/eShopModernizing>.
```

The project names are used throughout the eBook without emphasis, why change that now? This is something that should be discussed as a group and agreed on before hand. However, this is not code... either italic, bold, or both... or not at all.

Reply:
I see. Makes sense